go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mattermost/mattermost-load-test-ng
 
-go 1.20
+go 1.21
```

Comment:
Noting we are coupling this bump with the PR.

Reply:
Yeah, it's because the public package hence the server is bumped to `1.21`. Should we do it separately?

Reply:
Up to you but it may be cleaner to split.